### ISSUE TITLE
Request to Recruit Students Seeking Service Learning Hours

### ISSUE THREAD

**Issue by LPaulson1985** [ProgressiveCoders/functions#266]
#### Description

...
Request to recruit students seeking service learning hours to onboard and volunteer with Progcode projects. As a 501c3, Progcode is in a position to recruit and sign off on service learning hours students need to fulfill graduation requirements. Service learning opportunities will make Progcode an educational resource and provide students with learning and work experience, professional networking opportunities in a non-profit and/or technical setting, and experience contributing to start-up technical organizations aspiring to create positive impacts- all prior to stepping into an entry-level position in the professional world. This would also add another aspect of value to a very valuable organization and can expand our net for grant application, should the community decide to pursue funding.

#### Problem

...
Educational and professional mentoring
#### Benefit

...
Disparities in educational and employment opportunities are a direct impact of money in politics, which is largely corporate. Corporate politics invests in wealth, but overlooks the poor and diversity. So, they fall through the cracks, while the inherently wealthy get wealthier.

#### Plan

- [1] Discuss with Community and get Feedback
- [2] Revise as necessary
- [3] Review with Community
- [4] Bring to a community Vote
- [5] If approved, set up organizational accounts with Handshake, Idealist, and/or other community approved job boards 
- [6] Work through these platforms to build network relationships with schools and programs posting opportunities on educational facility-trusted sites that require organizational authentication (Handshake, etc). These can be linked to onboarding forms for application review and vetting process.
- [7] Attend career fairs
- [8] Host Virtual Organizational Events for the purpose of recruitment.
- [9] Review, Vet, and Vote on Applicants for Follow-up or dedicate a working committee to do so before bringing to a community vote
- [10] Onboard and mentor new service-learning students within the Progcode community.
- [11] Would like to start ASAP, but schedules can be unpredictable.
- 
...

#### Decision Making

...
It will take however long it takes.

#### Reference link(s)

...
Alex Kotch of OptOut can attest to how active student recruitment has been. I am happy to share Intro Student Presentation for OptOut if we decide to move forward. I have been using [Idealist](https://www.idealist.org/en/) at Jake Varghese's suggestion. and [Handshake]((https://www.google.com/search?channel=tus5&client=firefox-b-1-d&q=Handshake)). I have had a grat deal of response on HandShake from schools' career centers and students, primarily because it vets organizations for the schools and is directly linked to the schools service learning, career, and employment departments.

**Comment by jpb5013** [comment]
I'd definitely support this and also support allocating funding in the
budget toward paid opportunities like this.

**Comment by LPaulson1985** [comment]
Thank you, Joe! I am really glad to hear it and thankful for your support. I am of the mind that service-learning accreditation and student recruitment has the potential to strengthen current efforts on our very valuable, high-impact community projects and even further demonstrate the already existent, limitless organizational value of Progcode.

**Comment by LPaulson1985** [comment]
> A previous pilot initiative ([The Interns #206](https://github.com/ProgressiveCoders/functions/issues/206)) addressed whether and how to adopt an internship program. The plan Joe outlined there might supplement Lori's excellent plan in this proposal.

I love this idea! These are two distinct situations students run into. Can we include both internships and service-learning ops? I do suggest starting with service-learning, only because we need budget allocated to an internship program. But that may be more feasible than I am aware of. Either way, these are two services Progcode can provide that will increase educational opportunities in alignment with students educational and professional development goals and requirements that will increase accessibility and retain students that may not have had mobility or transportation, while also extending to diversity and international students who are often either passed over due to bigotry or exploited. If we can do any or all of these things-well funders love to see it. Education gets funding and diversity should as well. Very excited to carve this out together if the community is behind it.

**Comment by Sema-Hernandez** [comment]
I am aware the ProgCode community voted to proceed. What happens now? Is there a formal process in development to have long-term educational partnerships with grade schools, trade schools, colleges, and universities?
How are the credit hours verified for Service Learning Student?

I know this has been successful for OptOut, just wanted to know how this will be implemented in ProgCode.

I look forward to the exchange of ideas and formal process as we expand ProgCode activities.

**Comment by LPaulson1985** [comment]
@Sema-Hernandez The Handshake account was set up through the admin@progcode email, so until I am authorized or have a way to access that account, I cannot use Handshake to recruit. (2-Factor Authentication, etc) xxxx@xxxx.com, could add me with sufficient permission levels to perform recruitment and attend career fairs on ProgCode's behalf. If you know how to access xxxx@xxxx.com email, lmk. I had more authorization and range to do so with OptOut, so it was very simple. OptOut has been recruiting through Career Fairs rather than institutional partnerships. Credit Hours for students can be verified through time-tracking applications, like [Toggle](https://toggl.com/track/?utm_source=google&utm_medium=cpc&utm_campaign=Toggl%20Track%20-%20%5BS%5D%20-%20Global%20-%20Brand%20-%20Sign%20Up%20-%20Toggl%20Always%20On%20-%20Ad%203%20-%20Responsive&utm_term=time%20tracker&utm_content=search&cq_src=google_ads&cq_cmp=11493628079&cq_term=toggle&cq_plac=&cq_net=g&cq_plt=gp&gclid=CjwKCAjwuYWSBhByEiwAKd_n_nI-OTAq-jwhl0vZst_SwG4ztUCcT5weN14on1OkLU0O_GYLeB3DYRoC03sQAvD_BwE). My current Service-Learning student and I are tracking time through a simple time-tracking spreadsheet and documenting meetings in Google Calendar. Any documentation he needs signed, we can sign off on. However, there has currently been no documentation requested by his instructor. It's a real build it while you fly it situation, but has been great thus far. The long term plan that was drafted is outlined in the original github issue #266 post above.

**Comment by Sema-Hernandez** [comment]
@LPaulson1985, Has this been addressed with @jpb5013 or @stephenscapelliti to get access to the ProgCode admin email to access the Handshake account? And is the process going forward?


> @Sema-Hernandez The Handshake account was set up through the xxxx@xxxx.com email, so until I am authorized or have a way to access that account, I cannot use Handshake to recruit. (2-Factor Authentication, etc) Admin@progcode, could add me with sufficient permission levels to perform recruitment and attend career fairs on ProgCode's behalf. If you know how to access xxxx@xxxx.com email, lmk. I had more authorization and range to do so with OptOut, so it was very simple. OptOut has been recruiting through Career Fairs rather than institutional partnerships. Credit Hours for students can be verified through time-tracking applications, like [Toggle](https://toggl.com/track/?utm_source=google&utm_medium=cpc&utm_campaign=Toggl%20Track%20-%20%5BS%5D%20-%20Global%20-%20Brand%20-%20Sign%20Up%20-%20Toggl%20Always%20On%20-%20Ad%203%20-%20Responsive&utm_term=time%20tracker&utm_content=search&cq_src=google_ads&cq_cmp=11493628079&cq_term=toggle&cq_plac=&cq_net=g&cq_plt=gp&gclid=CjwKCAjwuYWSBhByEiwAKd_n_nI-OTAq-jwhl0vZst_SwG4ztUCcT5weN14on1OkLU0O_GYLeB3DYRoC03sQAvD_BwE). My current Service-Learning student and I are tracking time through a simple time-tracking spreadsheet and documenting meetings in Google Calendar. Any documentation he needs signed, we can sign off on. However, there has currently been no documentation requested by his instructor. It's a real build it while you fly it situation, but has been great thus far. The long term plan that was drafted is outlined in the original github issue #266 post above.

**Comment by LPaulson1985** [comment]
@Sema-Hernandez Yes, Stephen set the Handshake acct up this way for security. But even talking about these things in circles is so frustrating and exhausting. So many unreachable things. I can feel my blood pressure rising, bc it's literally been this circular and exhausting for so long. And I'm just tired. I'm honestly at that point of burnout.

**Comment by Sema-Hernandez** [comment]
@LPaulson1985,
Have you followed up on this to get access to the Handshake account with anyone?

> @Sema-Hernandez Yes, Stephen set the Handshake acct up this way for security. But even talking about these things in circles is so frustrating and exhausting. So many unreachable things. I can feel my blood pressure rising, bc it's literally been this circular and exhausting for so long. And I'm just tired. I'm honestly at that point of burnout.

**Comment by LPaulson1985** [comment]
@Sema-Hernandez No, after Stephen set up the Handshake student recruitment account with the xxxx@xxxx.com email, I did not get to follow up. That same week, I was doing Fundraising outreach as Justin asked when we were pursuing the Triangle Community Foundation grant for Enviro-Track, trying to get all the required consents to move forward so we could schedule a meeting with organizations and respond to the grant officer's offer to meet. This is why I am stepping back per our slack discussion with @jpb5013, concentrating on working where I can be effective, supporting my family, and narrowing my efforts.

**Comment by stephenscapelliti** [comment]
All ProgCode accounts are set up on e-mail accounts designated for that purpose. For that reason, I have edited the above comments wherever an e-mail account was referenced. We can discuss this more effectively and securely in the ProgCode workspace.
